UnityProject/Assets/Plugins/Zenject/Source/Runtime/Injection/InjectContext.cs

```diff
@@ -268,6 +268,28 @@ public string GetObjectGraphString()
                 result.AppendLine(context.ObjectType.PrettyName());
             }
 
+#if !NOT_UNITY3D
+            if (this.ObjectInstance is UnityEngine.Component component)
+            {
+                // Print the GameObject hierarchy
+                var transform = component.transform;
+                while (transform != null)
+                {
+                    if (transform.name.EndsWith("(Clone)"))
+                    {
+                        // Stop printing when we find an instantiated object, we are validating that prefab
+                        result.AppendLine(transform.name.Replace("(Clone)", " (prefab)"));
```

Comment:
I'm not sure if this will work with `WithGameObjectName` (it would overwrite the default name with "(Clone)"), but I guess it's not very critical.

Reply:
Yes, it'd print the whole hierarchy all the way to SceneContext but I think that's okay.